packages/meteor/client_environment.js

```diff
@@ -36,15 +37,32 @@ Meteor = {
    * @type {Boolean}
    */
   isServer: false,
-  isCordova: false
+
+  /**
+   * @summary Boolean variable.  True if running in Cordova environment.
+   * @locus Anywhere
+   * @static
+   * @type {Boolean}
+   */
+  isCordova: false,
+
+  /**
+   * @summary Boolean variable. True if running in a "modern" JS
+   *          environment, as determined by the `modern` package.
+   * @locus Anywhere
+   * @static
+   * @type {Boolean}
+   */
+  isModern: config.isModern
```

Comment:
Is this actually useful? Maybe for collecting stats at runtime about how many clients qualified for the "modern" treatment?